### ISSUE TITLE
Can i set the state of Toggle when initialize

### ISSUE THREAD

**Issue by programming2012** [yasirkula/UnitySimpleGDPRConsent#1]
Hi
   When i call AddSectionWithToggle, Is it possible to pass in a parameter to control the initial state of the Toggle? For example On(true) or Off(false).
   In addition, when I try to modify the content of the upper part of the Dialog, like:"Thank you for ... We aim..." . I did not find the method to modify it. Is there a quick way to modify it?

**Comment by yasirkula** [comment]
Hello,

You can run the following code before showing the GDPR consent dialog to default a setting to false:

```csharp
if( SimpleGDPRConsent.GDPRConsentCanvas.GetConsentState( "CONSENT_IDENTIFIER" ) == SimpleGDPR.ConsentState.Unknown )
	SimpleGDPRConsent.GDPRConsentCanvas.SetConsentState( "CONSENT_IDENTIFIER", SimpleGDPR.ConsentState.No );
```

But I agree that I should be adding a default value parameter in the next update.

For now, you can edit/localize the fixed parts of the consent canvas by modifying the **Plugin/SimpleGDPRConsent/Resources/GDPRConsentCanvas** prefab.

**Comment by programming2012** [comment]
Thank you very much for your reply, I am following these steps first.
I found this to be a time consuming operation when I called the SimpleGDPR.IsGDPRApplicable method. I tried using webClient.DownloadStringAsync or webClient.DownloadStringTaskAsync as an alternative, but I didn't get myself satisfied in the end. I will think about it later, if there is a good idea, then send a message.
                                                             Thanks
